Source/items.h

```diff
@@ -279,10 +279,6 @@ struct Item {
 	 */
 	bool isEquipment() const

```

Comment:
This one checks `_iLoc` rather than `_itype`.

Reply:
Yeah, I was wondering about that although _iLoc is being initialized to ILOC_NONE by default, so it should catch the default case, or am I wrong?

Reply:
Some areas of the code use `item.clear()` instead of `item = {}`.

https://github.com/diasurgical/devilutionX/blob/9e2608bd9855aaf40975d06d34521d4d3d063023/Source/items.h#L259-L265

Reply:
You're right, missed that, added back :)